buildconfig/Jenkins/Conda/conda-build-recipes

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash -ex
+
+# This script expects to be in a POSIX environment, the script clones the conda-recipes
+# repo and then runs conda builds based on passed args. By default it only clones the
+# source, you need to pass a flag to actually build something useful. If building a
+# package dependent on a different one it will build and produce that package too.
+#
+# Script usage:
+# conda-build-recipes WORKSPACE [options]
+#
+# Example command to build all of the packages:
+# conda-build-recipes $WORKSPACE --build-mantid --build-qt --build-workbench
+#
+# Expected args:
+#   1. WORKSPACE: path to the workspace/source code that this should run inside
+#                 (mantid repo). Windows Caveat: Only use / for this arguement do
+#                 not use \\ or \ in the path.
+#
+# Possible flags:
+#   --build-mantid: build the mantid package using conda-build
+#   --build-qt: build the qt package using conda-build
+#   --build-workbench: build the workbench package using conda-build
+#
+# Possible parameters:
+#   --clobber-yml: clobber the meta yaml files of each package build with conda-build, provide an absolute path to the uteyml file
+#
+
+# Setup expected variables
+WORKSPACE=$1
+shift
+if [[ $OSTYPE == "msys"* ]]; then
+    WORKSPACE=${WORKSPACE////\\}
+    # Require git settings so that paths can be checked out with longer file names than 260 characters
+    if [[ ! $(git config --system --get core.longpaths) == true ]]; then
+        echo "This windows machine is not configured properly, please run this command as an admin: git config --system core.longpaths true"
+        exit 1
+    fi
+fi
+EXPECTED_MAMBAFORGE_PATH=$WORKSPACE/mambaforge # Install into the WORKSPACE_DIR
```

Comment:
All instances of `$WORKSPACE` and it's derivatives should be quoted, to avoid escaping problems:
```bash
EXPECTED_MAMBAFORGE_PATH="$WORKSPACE/mambforge"